src/main/java/wooteco/subway/domain/station/Station.java

```diff
@@ -9,8 +8,7 @@
     @Id
     private Long id;
     private String name;
-    @CreatedDate
-    private LocalDateTime createdAt;
+    private LocalDateTime createdAt = LocalDateTime.now();
 
     public Station() {
```

Comment:
```suggestion
    private Station() {
```
객체에 불필요한 접근 방법은 막아주는게 좋습니다